packages/@aws-cdk/aws-apigatewayv2/lib/common/domain-name.ts

```diff
@@ -70,32 +66,19 @@ export interface DomainNameProps {
  */
 export class DomainName extends Resource implements IDomainName {
   /**
-   * import from attributes
+   * Import from attributes
    */
   public static fromDomainNameAttributes(scope: Construct, id: string, attrs: DomainNameAttributes): IDomainName {
     class Import extends Resource implements IDomainName {
       public readonly regionalDomainName = attrs.regionalDomainName;
       public readonly regionalHostedZoneId = attrs.regionalHostedZoneId;
-      public readonly domainName = attrs.domainName;
+      public readonly name = attrs.name;
     }
     return new Import(scope, id);
   }
 
-  /**
-   * The custom domain name for your API in Amazon API Gateway.
-   *
-   * @attribute
-   */
-  public readonly domainName: string;
-
-  /**
-   * The domain name associated with the regional endpoint for this custom domain name.
-   */
+  public readonly name: string;
```

Comment:
why did you remove the docs? Specifically, the `@attribute?`

Reply:
They are automatically inherited from the interface `IDomainName`.

Reply:
This is also why the linter does not complain.